### ISSUE TITLE
Sentinel Broken after 2.8.12

### ISSUE THREAD

**Issue by svantreeck** [microsoftarchive/redis#146]
After upgrading to the 2.8.12 binaries, my sentinels don't seem to connect anymore...the below is the output, where it appears that after 30s, I get an sdown notification.  Any ideas?

[11440] 08 Jul 12:49:35.235 # Sentinel runid is 28451df22550c981ae46210e3696250678cfcb5d
[11440] 08 Jul 12:49:35.236 # +monitor master mymaster 127.0.0.1 6379 quorum 2
[11440] 08 Jul 12:49:36.239 . +cmd-link master mymaster 127.0.0.1 6379
[11440] 08 Jul 12:49:36.241 . +pubsub-link master mymaster 127.0.0.1 6379
[11440] 08 Jul 12:49:51.389 . +cmd-link master mymaster 127.0.0.1 6379
[11440] 08 Jul 12:49:51.391 . +pubsub-link master mymaster 127.0.0.1 6379
[11440] 08 Jul 12:50:05.288 # +sdown master mymaster 127.0.0.1 6379
[11440] 08 Jul 12:50:06.481 . +cmd-link master mymaster 127.0.0.1 6379
[11440] 08 Jul 12:50:06.482 . +pubsub-link master mymaster 127.0.0.1 6379

My Sentinel Config looks like this:

```
maxheap 64MB
port 7379
dbfilename "sentinel-master.rdb"
loglevel debug

sentinel monitor mymaster 127.0.0.1 6379 2
sentinel down-after-milliseconds mymaster 30000
sentinel failover-timeout mymaster 180000
sentinel parallel-syncs mymaster 1
```

Master:

```
maxheap 256MB
port 6379
dbfilename "master.rdb"
maxmemory 256mb
maxmemory-policy allkeys-lru
```

Slave:

```
maxheap 256MB
port 6380
dbfilename "slave.rdb"
maxmemory 256mb
maxmemory-policy allkeys-lru
slaveof 127.0.0.1 6379
```

**Comment by jepickett** [comment]
I am looking at this. It may be a day or two before I have it fixed.

**Comment by jepickett** [comment]
Fixed in commit 4df3af4e6e80bfdbf6b937b6c281fc1e38a3cb70. Let me know if this works for you.

**Comment by svantreeck** [comment]
Looks good now, thanks!
